### ISSUE TITLE
Kurssipalautteeni view tab group keyboard navigation does not work

### ISSUE THREAD

**Issue by HRemonen** [UniversityOfHelsinkiCS/palaute#1233]
When focus is on the tab group the user should be able to navigate the tabs using arrow keys, now that's not the case.

Check full checklist of required functionality:
https://www.w3.org/WAI/ARIA/apg/patterns/tabs/
